front/src/app/components/common/benchmark/index.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { IBenchmarkMap, IDictionary } from 'app/api/types';
```

Comment:
you should define this types here if it is possible.

there are no any reason to place IDictionary typedef there. 
this definition is not related to api anyway